drivers/azure/azure.go

```diff
@@ -125,6 +130,18 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			EnvVar: "AZURE_SSH_USER",
 			Value:  defaultSSHUser,
 		},
+		mcnflag.StringFlag{
+			Name:   flAzureWinRMUser,
+			Usage:  "Username for WinRM login",
```

Comment:
let's prefix Usage for all new flags with `(Windows-only)` otherwise it'll be kind of confusing to linux machine users